aspnetcore/release-notes/aspnetcore-8.0.md

```diff
@@ -496,12 +496,6 @@ The antiforgery token is only validated if:
 
 For more information, see [Antiforgery with Minimal APIs](xref:security/anti-request-forgery?view=aspnetcore-8.0&preserve-view=true#afwma).
 
-### Support for `AsParameters` and automatic metadata generation
-
-Minimal APIs generated at compile-time include support for parameters decorated with the [`[AsParameters]`](xref:Microsoft.AspNetCore.Http.AsParametersAttribute) attribute and support automatic metadata inference for request and response types. Consider the following code:
-
-:::code language="csharp" source="~/release-notes/sample/ProgramAsParameters.cs" highlight="22":::
-
 The preceding generated code:
```

Comment:
Now it looks like you've got content talking about a snippet that is no longer there.